welcome/legal-notice.adoc

```diff
@@ -6,24 +6,6 @@ include::modules/common-attributes.adoc[]
 [.lead]
 Copyright © 2020 Red Hat, Inc.
 
-The text of and illustrations in this document are licensed by Red Hat under a Creative Commons Attribution–Share Alike 3.0 Unported license ("CC-BY-SA"). An explanation of CC-BY-SA is available at http://creativecommons.org/licenses/by-sa/3.0/. In accordance with CC-BY-SA, if you distribute this document or an adaptation of it, you must provide the URL for the original version. Modified versions must remove all Red Hat trademarks.
+OpenShift documentation is licensed under Apache License 2.0 (https://www.apache.org/licenses/).
 
-Red Hat, as the licensor of this document, waives the right to enforce, and agrees not to assert, Section 4d of CC-BY-SA to the fullest extent permitted by applicable law.
-
-Portions adapted from https://github.com/kubernetes-incubator/service-catalog/ with modifications by Red Hat. Licensed under the link:https://www.apache.org/licenses/LICENSE-2.0[Apache License 2.0].
```

Comment:
The sentence "Portions adapted from https://github.com/kubernetes-incubator/service-catalog/ with modifications by Red Hat." should (I'm assuming) be retained. We no longer need the sentence after it about the license.